inst/include/child_model_simulation.hpp

```diff
@@ -247,17 +255,16 @@ void convert_PMTCT_pre_bf(int t,
                           const State<ModelVariant, real_type> &state_curr,
                           State<ModelVariant, real_type> &state_next,
                           IntermediateData<ModelVariant, real_type> &intermediate) {
-  const auto cpars = pars.children.children;
   static_assert(ModelVariant::run_child_model,
                 "convert_PMTCT_num_to_perc can only be called for model variants where run_child_model is true");
-  constexpr auto hc_ss = StateSpace<ModelVariant>().children;
-  ///TODO: Maggie to confirm why Option A/B alt tr aren't used
-
-  for (int hp = 0; hp < hc_ss.hPS; ++hp) {
-    intermediate.children.PMTCT_coverage(hp) = (1 - cpars.PMTCT_transmission_rate(4,hp,0)) * intermediate.children.PMTCT_coverage(hp);
-
-  } //end hPS
-
+  constexpr auto ss_c = StateSpace<ModelVariant>().children;
+  const auto& p_hc = pars.children.children;
+  auto& i_hc = intermediate.children;
+
+  // TODO: Maggie to confirm why Option A/B alt tr aren't used
+  for (int hp = 0; hp < ss_c.hPS; ++hp) {
+    i_hc.PMTCT_coverage(hp) *= 1 - p_hc.PMTCT_transmission_rate(4, hp, 0);
```

Comment:
point of interest: small change in algebra using `x *= y` instead of `x = x * y`